### DISCUSSION TITLE
关于杜比视界/全景声/音效的选取逻辑

### DISCUSSION THREAD

**Discussion by SigureMo** [yutto-dev/yutto#62] [General]
目前 B 站是将杜比视界（Dolby Vision）放在了视频清晰度选择栏中，而杜比音效（Dolby Audio）和杜比全景声（Dolby Atmos）则是单独设一个按钮去选择是否开启。

<img width="910" alt="image" src="https://user-images.githubusercontent.com/38436475/154354419-8a91cc92-331c-416d-a021-15169c7befec.png">

> 图为 macOS 上 Safari 的截图，Chrome 貌似上还不能开启 Dolby，视频为 BV1eV411W7tt

目前 yutto 是将杜比视界的按照 B 站逻辑直接与清晰度混在一起了（4K 与 8K 之间），而声音的还没有做。唔，主要是不清楚怎么做合适，这个可以按照 B 站的逻辑，设置一个 Option，只有开启这个 Option 的情况下才会下载 Dolby 的音效。不过这样一来我觉得是不是杜比视界也可以从清晰度中分离出来，杜比视界这个东西本身和清晰度是挂钩的吗？它不是只是与色彩相关吗？感觉 B 站上的选取逻辑好奇怪啊……唔，关于音视频这些我都不太懂，希望大家能给出建议～

**Comment by goodryb** [comment]
这两种都需要播放设备支持才能有效果，不然可能效果还不如SDR版本，个人感觉意义不大。

**Reply by SigureMo** [reply]
> 这两种都需要播放设备支持才能有效果，不然可能效果还不如SDR版本

嗯对啊，所以我的想法是把这个单独提出来做一个选项，就不把它作为一种「清晰度」，不然的话用户可能并不需要杜比什么的。

> 个人感觉意义不大。

这我不明白，你说的是什么意义不大，支持杜比吗？那我觉得有问题，支持还得支持，只不过支持的方式需要去详细考量
